eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -22,7 +22,7 @@ steps:
       displayName: Build the ILC and RyuJit cross-compilers
 
     - ${{ if eq(parameters.isOfficialBuild, true) }}:
-      - script: $(Build.SourcesDirectory)/build$(scriptExt) libs+nativeaot.packages -c $(buildConfigUpper)
+      - script: $(Build.SourcesDirectory)/build$(scriptExt) libs+nativeaot.packages -c $(buildConfigUpper) /p:OfficialBuildId=$(Build.BuildNumber)
```

Comment:
The build number is included in all binaries as version resource. Should it be passed in for "Build the ILC and RyuJit cross-compilers" step as well?